examples/threathunter/threat_intelligence/config.yml

```diff
@@ -0,0 +1,70 @@
+sites:
+    alienvault:
+        # the feed_id of the ThreatHunter Feed you want to send ThreatIntel to
+        feed_id: DpB1mJvSc2ONzPjRm9BvA
+
+        # the address of the site (only server ip or dns; don't put https:// or a trailing slash)
+        site: otx.alienvault.com
+
+        # the path of the site for discovering what services are available
+        # this is supplied by your taxii provider
+        discovery_path: /taxii/discovery
+
+        # the path of the site for listing what collections are available to you
+        # this is supplied by your taxii provider
+        collection_management_path: /taxii/collections
+
+        # the path of the site for polling a collection
+        # this is supplied by your taxii provider
+        poll_path: /taxii/poll
+
+        # change to true if you require https for your TAXII service connection
+        use_https: True
+
+        # by default, we validate SSL certificates. Turn this off by setting ssl_verify: False
+        ssl_verify: False
+
+        # (optional) if you need SSL certificates for authentication, set the path of the
+        # certificate and key here. Please leave blank to ignore.
+        cert_file:
+        key_file:
+
+        # how to score each result. Accepts values [1,10], and defaults to 5
+        default_score: 5
+
+        # (optional) username for authorization with your taxii provider
+        username:
+
+        # (optional) password for authorization with your taxii provider
+        password:
+
+        # (optional) specify which collections to convert to feeds (comma-delimited)
+        collections: '*'
+
+        # (optional) the start date for which to start requesting data.
+        # Defaults to 2019-01-01 00:00:00 if you supply nothing
+        start_date: 2019-01-01 00:00:00
+
+        # (optional) the minutes to advance for each request.
+        # If you don't have a lot of data, you could advance your requests
+        # to every 60 minutes, or 1440 minutes for daily chunks
+        # defaults to 60
+        minutes_to_advance: 1440
+
+        # (optional) path to a CA SSL certificate
+        ca_cert:
+
+        # (optional) if you need requests to go through a proxy, specify an http URL here
+        http_proxy_url:
+
+        # (optional) if you need requests to go through a proxy, specify an https URL here
+        https_proxy_url:
+
+        # (optional) number of reports to collect from each site
+        # defaults to 10K
+        reports_limit: 10000
```

Comment:
Is there an option for no limit? Like putting 0 would disable the limit?

Reply:
I've updated stix_taxii.py so that now a user can have no report limit. It's also explained in the config